panda-qa/Resources/CommonFunctionality.robot

```diff
@@ -84,16 +84,14 @@ Start Barrons Article
     Wait Until Page Contains Element  dom:${JSFollowButtonBarronsPath}
     Maximize Browser Window
 
-
 Start Mansion Global Article
-    Set Selenium Speed  1 seconds
+    Set Selenium Speed  0.5 seconds
     ${options} =  Set Browser Options
     IF  "${Env}" == "prod"
-        Open Browser  https://www.mansionglobal.com  ${Browser}  options=${options}
+        Open Browser  https://www.mansionglobal.com  ${Browser}
         Go To  https://www.mansionglobal.com/articles/are-there-tax-breaks-on-agricultural-land-in-pennsylvania-01648119848
     ELSE IF  "${Env}" == "dev"
-        SeleniumLibrary.Open Browser  https://www.s.dev.mansionglobal.com  ${Browser}  options=${options}
-        Open Browser  https://www.s.dev.mansionglobal.com  ${Browser} options=${options}
+        Open Browser  https://www.s.dev.mansionglobal.com  ${Browser}
```

Comment:
do not remove   options=${options} at the end. please update it everywhere you changed.